tests/common.go

```diff
@@ -6624,3 +6625,123 @@ func AddMetadataDisk(n node.Node) error {
 	return nil
 
 }
+
+// generateRandomString Creates a random string of length n and returns it
+func generateRandomString(n int) string {
```

Comment:
common.go contains a function called RandomString which generates a random string of lowercase characters with a length specified by the length parameter.

Reply:
Okay let's remove it.